db-scripts/src/main/resources/cgds.sql

```diff
@@ -123,6 +123,7 @@ CREATE TABLE `cancer_study` (
   `GROUPS` varchar(200) DEFAULT NULL,
   `STATUS` int(1) DEFAULT NULL,
   `IMPORT_DATE` datetime DEFAULT NULL,
+  `METADATA` text,
```

Comment:
@pieterlukasse Should we validate the maximum size of the TEXT field type?

Reply:
@sandertan not really necessary, as long as the data loading step crashes with a clear error. @oplantalech can you test this with e.g. a file of 100KB?